src/core/src/bootstrap/Constants.py

```diff
@@ -199,6 +199,7 @@ class AutoAssessmentStates(EnumBackport):
     MAX_INSTALLATION_RETRY_COUNT = 3
     MAX_IMDS_CONNECTION_RETRY_COUNT = 5
     MAX_ZYPPER_REPO_REFRESH_RETRY_COUNT = 5
+    PATCHING_BATCH_SIZE = 3
```

Comment:
suggestion: This could be MAX_BATCH_SIZE_FOR_PACKAGES

Reply:
Updated to MAX_BATCH_SIZE_FOR_PACKAGES